jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh

```diff
@@ -5,7 +5,7 @@ IS_HEAT=_NOTHEAT
 
 if [ "$IS_HEAT" = HEAT ]; then
   # When run by Heat template, these will be substituted with the template's parameters
-  TESTHEAD=_TESTHEAD
+  OSHEAD=_TESTHEAD
```

Comment:
`OSHEAD` is used in jenkins/ci.opensuse.org/openstack-cleanvm.yaml . Also what's wrong with `OSHEAD`?

Reply:
Also OSHEAD is used in qa_openstack.sh which is called below and why we set it here. Nothing is wrong with OSHEAD, it is just not named TESTHEAD, which is what this change corrects.